src/libraries/Common/src/System/Security/Cryptography/RsaPaddingProcessor.cs

```diff
@@ -182,20 +163,22 @@ internal static RsaPaddingProcessor OpenProcessor(HashAlgorithmName hashAlgorith
             source.CopyTo(destination.Slice(paddingLength + 3 + digestInfoPrefix.Length));
         }
 
-        internal void PadOaep(
+        internal static void PadOaep(
+            HashAlgorithmName hashAlgorithmName,
             ReadOnlySpan<byte> source,
             Span<byte> destination)
         {
             // https://tools.ietf.org/html/rfc3447#section-7.1.1
 
+            GetDigestInfoForAlgorithm(hashAlgorithmName, out int hLen);
```

Comment:
```suggestion
            int hLen = HashLength(hashAlgorithmName);
```

Looks a bit more straightforward to me